compose.py

```diff
@@ -315,9 +315,12 @@ def create_argmap(images):
         'Use specified Docker image to generate Docker image. Specified as <image-name>,<full-image-name>. <image-name> can be "min" or "full". Both "min" and "full" need to be specified at the same time. This will override "--container-version".'
     )
     parser.add_argument('--enable-gpu',
```

Comment:
Does this allow both --enable-gpu and --enable-gpu=true/false? We need to accept --enable-gpu for backwards compat.

Reply:
@deadeyegoodwin both works. Please see CI run :)